editoast/editoast_models/src/rolling_stock/rolling_stock_category.rs

```diff
@@ -0,0 +1,84 @@
+use std::io::Write;
+use std::str::FromStr;
+
+use diesel::deserialize::FromSql;
+use diesel::deserialize::FromSqlRow;
+use diesel::expression::AsExpression;
+use diesel::pg::Pg;
+use diesel::pg::PgValue;
+use diesel::serialize::Output;
+use diesel::serialize::ToSql;
+use serde::Deserialize;
+use serde::Serialize;
+use strum::Display;
+use strum::EnumString;
+use strum::IntoStaticStr;
+use utoipa::ToSchema;
+
+editoast_common::schemas! {
+    RollingStockCategory,
+    RollingStockCategories,
+}
+
+/// This enum maps to a Postgres enum type, specifically `rolling_stock_category`.
+/// Any changes made to this enum must be reflected in the corresponding Postgres enum,
+/// and vice versa, to ensure consistency between the application and the database.
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    Serialize,
+    Deserialize,
+    ToSchema,
+    FromSqlRow,
+    AsExpression,
+    EnumString,
+    IntoStaticStr,
+    Display,
+)]
+#[diesel(sql_type = crate::tables::sql_types::RollingStockCategory)]
+#[strum(serialize_all = "kebab-case")]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum RollingStockCategory {
+    Unknown,
+    HighSpeedTrain,
+    IntercityTrain,
+    RegionalTrainMU,
```

Comment:
What does `MU` stands for? I'd suggest to respect CamelCase even for acronyms:

```suggestion
    RegionalTrainMu,
```

Reply:
I believe it stands for "Multiple Unit"?

Reply:
Done.